docs/secure-connections/sauce-connect/system-requirements.md

```diff
@@ -16,7 +16,7 @@ Keep in mind that these are guidelines. Most environments have their own workloa
 
 - Minimum 2 core x 8 GB Machine.
 - We recommend using the [cURL command line](https://curl.haxx.se/download.html) or an equivalent tool to ensure that any error messages you receive are precise and actionable. If you're not familiar with the tool, [review their docs here](https://curl.se/docs) and then install cURL on your local machine.
-- For Unix-based systems, you may need to increase your [open file limit](https://www.tecmint.com/increase-set-open-file-limits-in-linux/) to, at least, 64000.
+- For Unix-based systems, set your [open file limit](https://www.tecmint.com/increase-set-open-file-limits-in-linux/) to at least 64000.
```

Comment:
I'd suggest adding an article to SC FAQ page and link it here. Smth like this (feel free to change the content, it's just a quick example):

>>>>>>>>
What are my optimal open file limit settings?

Open file limit on macOS/Linux systems limits the number of open file descriptors. Despite what the name suggests, it does not just limit the number of files but also the number of open connections. Under normal usage, SC may easily hit the default limit for some systems of 1024 connections and, due to this, each new connection would time out, which results in failed tests. Modern systems allow much higher limits without any visible impact on memory and therefore we recommend setting the limit to a significantly higher value of 64000 in order to ensure the file limit is never an issue for the tunnelled requests.